indra/newview/llpuppetmotion.cpp

```diff
@@ -803,24 +797,26 @@ void LLPuppetMotion::solveIKAndHarvestResults(const LLIK::Solver::joint_config_m
 
 	LLPuppetEvent broadcast_event;
     const LLIK::Solver::joint_list_t& active_joints = mIKSolver.getActiveJoints();
-    LLVector3 pos;
-    LLQuaternion rot;
     for (auto joint: active_joints)
     {
         S16 id = joint->getID();
-        U8 flags = joint->getConfigFlags();
-        pos = joint->getLocalPos();
-        rot = joint->getLocalRot();
```

Comment:
Below we are figuring out what exact properties have changed so we can apply them to our avatar (`local_puppetry`) and/or stream to the server (`is_sending`).